src/components/Email/EmailTemplates/AdminInviteEmail.jsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable global-require */
+/* eslint-disable react/jsx-no-target-blank */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Email, Item } from 'react-html-email';
+import { Image } from '@chakra-ui/react';
+import { Link } from 'react-router-dom';
+
+import OCHLogo from '../../../assets/OCHlogo.png';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'OCHLogo' is defined but never used.